src/shared/ui/survey/CheckBox/CheckBoxItem.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, useState } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { colors } from '@shared/lib';
+import {
+  Image,
+  Text,
+  XStack,
+  QuestionTooltipIcon,
+  Tooltip,
+  CheckBox,
+} from '@shared/ui';
+
+import { Item } from './types';
+import { invertColor } from './utils';
+
+const handleReplaceBehaviourResponse = (string: string) => string;
+
+type CheckBoxItemProps = {
+  item: Item;
```

Comment:
It seems that instead of `Item` type we need here "color", "image", "description", "value" and "name" props on the same level